akka-docs/src/main/paradox/remoting.md

```diff
@@ -271,22 +271,22 @@ Scala
 Java
 :   @@snip [RemoteDeploymentDocTest.java](/akka-docs/src/test/java/jdocs/remoting/RemoteDeploymentDocTest.java) { #deploy }
 
-### Remote deployment whitelist
+### Remote deployment allow list
 
-As remote deployment can potentially be abused by both users and even attackers a whitelist feature
+As remote deployment can potentially be abused by both users and even attackers an allow list feature
```

Comment:
maybe native speakers will be able to tell it better, but I think we should called it `allowed-list` instead of `allow list`. 

@chbatey / @seglo ☝️

Reply:
The point being, "allow" is a verb so we should use its participle form to make it an adjective.

Reply:
I found some previous usage of "allow list" in various places so I think that is fine (here for example https://www.zdnet.com/article/uk-ncsc-to-stop-using-whitelist-and-blacklist-due-to-racial-stereotyping/ ).

Reply:
I think `allow` is fine as well. I'm not sure why you would use the past participle for `allow` and not `deny`, unless you were suggesting doing the same for `deny`.  Anecdotally, I've usually seen it as `allow`/`deny`.

I'm no English major though ;)

Reply:
It was implicit that I was suggesting to change `deny` as well. I find it strange to use a verb in the place of an adjective without making it a participle. 

But fair enough, it may be fine if you read it as: "we allow/deny what is in that list". I will get use to.